src/core/lib/slice/slice.h

```diff
@@ -103,6 +104,10 @@ class BaseSlice {
     return grpc_slice_is_equivalent(slice_, other.slice_);
   }
 
+  uint32_t Hash() const {
+    return gpr_murmur_hash3(data(), length(), g_hash_seed);
```

Comment:
Should this call `grpc_slice_hash_internal()`, just so that we don't have two different hash functions that might diverge?

Reply:
reasonable